tickerService.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"time"
+)
+
+type Ticker interface {
+	Tick(device DeviceWithId, tickerService TickerService)
+}
+
+type TickerService struct {
+	DeviceService DeviceService
+	measurement   chan Measurement
+	isRunning     bool
+	stop          chan struct{}
+	Ticker
+}
+
+type MeasurementTicker struct{}
+
+type Measurement struct {
+	Id    int
+	Value float64
+}
+
+func (ts *TickerService) Start() (started bool, error error) {
+	if !ts.isRunning {
+		ts.isRunning = true
+		deviceWithId, err := ts.DeviceService.Get(0, 0)
+		if err != nil {
+			return false, err
+		}
+		for _, device := range deviceWithId {
+			go ts.Ticker.Tick(device, *ts)
+		}
+		return true, nil
+	}
+	return false, nil
+}
+
+func (ts *TickerService) Stop() (stopped bool, error error) {
+	if ts.isRunning {
+		ts.isRunning = false
+		close(ts.stop)
+		return true, nil
+	}
+	return false, nil
+}
+
+func (mt *MeasurementTicker) Tick(device DeviceWithId, ts TickerService) {
```

Comment:
I think that this method can be private and be part of `TickerService`, right?

Reply:
`MeasurementTicker` is separate struct because of tests right? if yes then let's leave it like that